genny-project/layouts
# Feature intm 832

# Issue https://outcomelife.atlassian.net/browse/INTM-832

# Description:
Refactoring layouts for induction and agreement documents.

# How To Test
1) Go to Internmatch.
2) Create an Internship
3) Create an Application and move it through the process until it is in the Offered column.
4) Click on the card, and view the Induction document. Should go to new page.
5) View the Agreement. Should go to new page.
